### ISSUE TITLE
Splash Witches' Brew of Fertility causes server crash

### ISSUE THREAD

**Issue by joakibj** [Um-Mitternacht/Bewitchment#462]
### Expected Behavior
Throwing Splash Witches' Brew (Fertility) on a grass block, being able to grow grass just like applying bone meal.

### Actual Behavior
Throwing Splash Witches' Brew (Fertility) on a grass block causes the server to crash.

Consequent attempts to enter the world also results in the same server crash.

Was able to get in to the world after 9 attempts.

### Steps to Reproduce

Brew a Splash Witches' Brew (Fertility).

1. Mandrake Root
2. Gunpowder
3. Blue Orchid

Bottle the brew.

Throw brewed potion on grass block.

### Version of Minecraft, Bewitchment, Forge, Patchouli, and Baubles used
```
Minecraft 1.12.2
Forge 14.23.5.2847
Baubles-1.12-1.5.2
bewitchment-1.12.2-0.0.21.1
jei_1.12.2-4.15.0.291
Patchouli-1.0-20
```

### Logs, in the event of a crash
Server crash log:
```
---- Minecraft Crash Report ----

WARNING: coremods are present:
  BewitchmentFMLLoadingPlugin (bewitchment-1.12.2-0.0.21.1.jar)
Contact their authors BEFORE contacting forge

// Who set us up the TNT?

Time: 12/4/19 7:46 PM
Description: Ticking entity

java.lang.IllegalArgumentException: Cannot get property PropertyEnum{name=type, clazz=class net.minecraft.block.BlockTallGrass$EnumType, values=[dead_bush, tall_grass, fern]} as it does not exist in BlockStateContainer{block=minecraft:double_plant, properties=[facing, half, variant]}
	at net.minecraft.block.state.BlockStateContainer$StateImplementation.func_177229_b(BlockStateContainer.java:201)
	at net.minecraft.block.BlockTallGrass.func_176474_b(BlockTallGrass.java:101)
	at com.bewitchment.common.potion.PotionFertility.onImpact(PotionFertility.java:45)
	at net.minecraft.entity.projectile.EntityPotion.handler$applyEffects$zza000(SourceFile:526)
	at net.minecraft.entity.projectile.EntityPotion.func_70184_a(SourceFile)
	at net.minecraft.entity.projectile.EntityThrowable.func_70071_h_(EntityThrowable.java:249)
	at net.minecraft.world.World.func_72866_a(World.java:1996)
	at net.minecraft.world.WorldServer.func_72866_a(WorldServer.java:832)
	at net.minecraft.world.World.func_72870_g(World.java:1958)
	at net.minecraft.world.World.func_72939_s(World.java:1762)
	at net.minecraft.world.WorldServer.func_72939_s(WorldServer.java:613)
	at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:767)
	at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:668)
	at net.minecraft.server.integrated.IntegratedServer.func_71217_p(IntegratedServer.java:185)
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:526)
	at java.lang.Thread.run(Thread.java:748)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Server thread
Stacktrace:
	at net.minecraft.block.state.BlockStateContainer$StateImplementation.func_177229_b(BlockStateContainer.java:201)
	at net.minecraft.block.BlockTallGrass.func_176474_b(BlockTallGrass.java:101)
	at com.bewitchment.common.potion.PotionFertility.onImpact(PotionFertility.java:45)
	at net.minecraft.entity.projectile.EntityPotion.handler$applyEffects$zza000(SourceFile:526)
	at net.minecraft.entity.projectile.EntityPotion.func_70184_a(SourceFile)
	at net.minecraft.entity.projectile.EntityThrowable.func_70071_h_(EntityThrowable.java:249)
	at net.minecraft.world.World.func_72866_a(World.java:1996)
	at net.minecraft.world.WorldServer.func_72866_a(WorldServer.java:832)
	at net.minecraft.world.World.func_72870_g(World.java:1958)

-- Entity being ticked --
Details:
	Entity Type: minecraft:potion (net.minecraft.entity.projectile.EntityPotion)
	Entity ID: 1900887
	Entity Name: Potion
	Entity's Exact location: 1107.70, 4.97, 740.50
	Entity's Block location: World: (1107,4,740), Chunk: (at 3,0,4 in 69,46; contains blocks 1104,0,736 to 1119,255,751), Region: (2,1; contains chunks 64,32 to 95,63, blocks 1024,0,512 to 1535,255,1023)
	Entity's Momentum: -0.23, -0.26, 0.41
	Entity's Passengers: []
	Entity's Vehicle: ~~ERROR~~ NullPointerException: null
Stacktrace:
	at net.minecraft.world.World.func_72939_s(World.java:1762)
	at net.minecraft.world.WorldServer.func_72939_s(WorldServer.java:613)

-- Affected level --
Details:
	Level name: New World
	All players: 1 total; [EntityPlayerMP['xyz'/19, l='New World', x=1109.10, y=4.00, z=738.03]]
	Chunk stats: ServerChunkCache: 625 Drop: 0
	Level seed: 1083729175757474886
	Level generator: ID 01 - flat, ver 0. Features enabled: true
	Level generator options: 
	Level spawn location: World: (1173,4,736), Chunk: (at 5,0,0 in 73,46; contains blocks 1168,0,736 to 1183,255,751), Region: (2,1; contains chunks 64,32 to 95,63, blocks 1024,0,512 to 1535,255,1023)
	Level time: 11750 game time, 11750 day time
	Level dimension: 0
	Level storage version: 0x04ABD - Anvil
	Level weather: Rain time: 79126 (now: false), thunder time: 119832 (now: false)
	Level game mode: Game mode: creative (ID 1). Hardcore: false. Cheats: true
Stacktrace:
	at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:767)
	at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:668)
	at net.minecraft.server.integrated.IntegratedServer.func_71217_p(IntegratedServer.java:185)
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:526)
	at java.lang.Thread.run(Thread.java:748)

-- System Details --
Details:
	Minecraft Version: 1.12.2
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 1.8.0_222, AdoptOpenJDK
	Java VM Version: OpenJDK 64-Bit Server VM (mixed mode), 
	Memory: 673407728 bytes (642 MB) / 1204023296 bytes (1148 MB) up to 4207738880 bytes (4012 MB)
	JVM Flags: 23 total; -XX:+DisableExplicitGC -XX:+UseConcMarkSweepGC -XX:+UseNUMA -XX:+CMSParallelRemarkEnabled -XX:MaxTenuringThreshold=15 -XX:MaxGCPauseMillis=30 -XX:GCPauseIntervalMillis=150 -XX:+UseAdaptiveGCBoundary -XX:-UseGCOverheadLimit -XX:+UseBiasedLocking -XX:SurvivorRatio=8 -XX:TargetSurvivorRatio=90 -XX:MaxTenuringThreshold=15 -XX:+UseCompressedOops -XX:+OptimizeStringConcat -XX:+AggressiveOpts -XX:ReservedCodeCacheSize=2048m -XX:+UseCodeCacheFlushing -XX:SoftRefLRUPolicyMSPerMB=10000 -XX:ParallelGCThreads=10 -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xms1024m -Xmx4096m
	IntCache: cache: 0, tcache: 0, allocated: 0, tallocated: 0
	FML: MCP 9.42 Powered by Forge 14.23.5.2847 8 mods loaded, 8 mods active
	States: 'U' = Unloaded 'L' = Loaded 'C' = Constructed 'H' = Pre-initialized 'I' = Initialized 'J' = Post-initialized 'A' = Available 'D' = Disabled 'E' = Errored

	| State  | ID          | Version      | Source                                  | Signature |
	|:------ |:----------- |:------------ |:--------------------------------------- |:--------- |
	| LCHIJA | minecraft   | 1.12.2       | minecraft.jar                           | None      |
	| LCHIJA | mcp         | 9.42         | minecraft.jar                           | None      |
	| LCHIJA | FML         | 8.0.99.99    | forge-1.12.2-14.23.5.2847-universal.jar | None      |
	| LCHIJA | forge       | 14.23.5.2847 | forge-1.12.2-14.23.5.2847-universal.jar | None      |
	| LCHIJA | baubles     | 1.5.2        | Baubles-1.12-1.5.2.jar                  | None      |
	| LCHIJA | patchouli   | 1.0-20       | Patchouli-1.0-20.jar                    | None      |
	| LCHIJA | bewitchment | 0.21.1       | bewitchment-1.12.2-0.0.21.1.jar         | None      |
	| LCHIJA | jei         | 4.15.0.291   | jei_1.12.2-4.15.0.291.jar               | None      |

	Loaded coremods (and transformers): 
BewitchmentFMLLoadingPlugin (bewitchment-1.12.2-0.0.21.1.jar)
  
	GL info: ~~ERROR~~ RuntimeException: No OpenGL context found in the current thread.
	Profiler Position: N/A (disabled)
	Player Count: 1 / 8; [EntityPlayerMP['xyx'/19, l='New World', x=1109.10, y=4.00, z=738.03]]
	Type: Integrated Server (map_client.txt)
	Is Modded: Definitely; Client brand changed to 'fml,forge'
```

**Comment by Sunconure11** [comment]
Possibly fixed.

**Comment by joakibj** [comment]
Confirmed fixed in 52201c48.
